### ISSUE TITLE
Native Application Not Found

### ISSUE THREAD

**Issue by JamesDalboth** [Frewacom/Pywalfox#14]
I am currently experiencing issues using the extensions. After a firefox reboot the browser console shows 

![2020-03-23_222909-1xMar_scrot](https://user-images.githubusercontent.com/21280865/77451056-0f9edb80-6dec-11ea-88c6-ded965b08f31.png)

The relevant line being `No such native application pywalfox`

My python version is 3.6.9.

My manifest is located in `~/.mozila/native-messaging-hosts/pywalfox.json` and looks like

```
{
  "name": "pywalfox",
  "description": "Automatically theme firefox using system colors generated by Pywal",
  "path": "/home/james/Pywalfox/daemon/pywalfox.py",
  "type": "stdio",
  "allowed_extensions": [ "pywalfox@frewacom.org" ]
}
```

I can confirm the `pywalfox.py` is located in the specified path param. The file has read privileges for all. I have tried to relocate the file to the other specified native manifest locations specified in the moz docs.

**Comment by Frewacom** [comment]
What OS are you running?

**Comment by JamesDalboth** [comment]
Ubunutu 18.04

**Comment by JamesDalboth** [comment]
Update: Literally just got it working. I read somewhere about a problem with using firefox-bin. Can't remember how I installed firefox originally but I saw in my process manager I was using firefox-bin. I just tried installing firefox74 directly from the mozilla page. Process mananger still shows it running as firefox-bin however it now works. Problem could of simply been my previous firefox version. I will see If I can find what version I was previously running.
